### DISCUSSION TITLE
KeyError when training the tagger

### DISCUSSION THREAD

**Discussion by JessitaMS** [explosion/spaCy#10601] [Help: Installation]
I am presenting the following KeyError: '' when I am going to train the model with the data that I provide.

The data has the following format:
Word POS Tag

I then convert them to iob, then to spacy.
Then I configure the config so that within the pipeline I include tagger, ner.

And when it is going to execute it gives me this error, I don't know why it is not taking the POS values ​​that go in the input dataset.

I have found another issue similar to this one, for now no link of what is provided is active, for this reason I have reopened the discussion

**Comment by polm** [comment]
This is a duplicate of #10590. Don't post duplicate threads. I'll lock this to keep discussion in one place.
